src/common/sound.cpp

```diff
@@ -130,19 +132,32 @@ void Sound::stop() {
 
 void Sound::_playSound(eSOUND_TYPE sound, const eSOUND_TYPE types[],
```

Comment:
These changes (for sounds ignoring volume and more) is addressed also in #994. Please revert your commit or take a good care when merging.